.vscode/settings.json

```diff
@@ -27,6 +27,7 @@
     // testing
     "python.testing.unittestArgs": [
         "-v",
+        "-s",
```

Comment:
What does this flag do? Unless I comment it out VSCode doesn't find the tests.

Reply:
Looks like it specifies the directory for tests. It should've been something like this:

<img width="778" alt="image" src="https://github.com/ctsit/report_handler/assets/43610780/6f7d4fa4-5430-4a00-bb66-776c40ac2ad1">


I will fix the tests and add this change to the commit also.